.travis.yml

```diff
@@ -24,10 +24,7 @@ notifications:
 
 before_install:
   - gem update bundler
-  - .travis/install_gradle.sh
-  - .travis/install_rebar.sh
-  - .travis/install_bower.sh
-  - .travis/install_godep.sh
+  - for s in .travis/install_*.sh; do $s; done
```

Comment:
This is more concise, but doesn't clarify what's getting run, and makes assumptions (perhaps OK) that ordering doesn't matter. I'm +0 for this change.